frontend/components/commons/sidebar-feedback-task/SidebarFeedbackTaskProgress.vue

```diff
@@ -90,6 +103,15 @@ export default {
       return this.totalResponded / this.progressTotal;
     },
   },
+  methods: {
+    capitalize(text) {
+      const baseText = text.toLowerCase();
+      return `${baseText.charAt(0).toUpperCase()}${baseText.slice(1)}`;
+    },
```

Comment:
this kind of global function should be external of the component

Reply:
yes, I have replaced it with a existed filter